overlord/auth/auth.go

```diff
@@ -64,6 +64,16 @@ type UserState struct {
 	StoreDischarges []string `json:"store-discharges,omitempty"`
 }
 
+// idOnly returns a *UserState with only the identification information
+// from u.
+func (u *UserState) idOnly() *UserState {
```

Comment:
This looks fine though on first look I assumed it would return `uid`. Still, because this is private I don't mind at all.